src/features/profile/HouseKeepingCard.js

```diff
@@ -29,14 +29,14 @@ const HouseKeepingCard = () => {
   const [isPending, setIsPending] = useState(false)
   const entries = useSelector(selectEntries)
```

Comment:
Here you have all the entries, when the user click the export these entries shall be transformed into an csv file.